source/whats-new.txt

```diff
@@ -33,6 +33,15 @@ Learn what's new in:
 What's New in 1.11
 --------------------
 
+.. important:: Upgrade to Version 1.11.2 or Higher 
+
+  The 1.11.2 Go driver contains the following bug fixes:
+  
+  - ``bson.Raw.String`` outputs the correct Extended JSON type for timestamps when 
```

Comment:
Suggestion: from what I found in the API docs, this is referring to a method. I suggest including the parentheses to make this clearer, or going even further and describing it as a method

```suggestion
  - ``bson.Raw.String()`` outputs the correct Extended JSON type for timestamps when 
```
or
```suggestion
  - The ``bson.Raw.String()`` method outputs the correct Extended JSON type for timestamps when 
```